Mage.Sets/src/mage/cards/v/VirtueOfCourage.java

```diff
@@ -0,0 +1,92 @@
+package mage.cards.v;
+
+import mage.abilities.TriggeredAbilityImpl;
+import mage.abilities.effects.common.DamageTargetEffect;
+import mage.abilities.effects.common.ExileTopXMayPlayUntilEndOfTurnEffect;
+import mage.cards.AdventureCard;
+import mage.cards.CardSetInfo;
+import mage.constants.CardType;
+import mage.constants.TargetController;
+import mage.constants.Zone;
+import mage.filter.FilterStackObject;
+import mage.game.Game;
+import mage.game.events.DamagedPlayerEvent;
+import mage.game.events.GameEvent;
+import mage.target.common.TargetAnyTarget;
+
+import java.util.UUID;
+
+/**
+ * @author Susucr
+ */
+public final class VirtueOfCourage extends AdventureCard {
+
+    private static final FilterStackObject filter
```

Comment:
what's up with this? seems irrelevant

Reply:
Yup, looks like I did start by copying another card, and forgot to cleanup that filter.